### PR TITLE
Decrease epochs for hyperparam example tests

### PR DESCRIPTION
**Pull request by harupy**
Signed-off-by: harupy <17039389+harupy@users.noreply.github.com>

## What changes are proposed in this pull request?

Decrease epochs for the hyperparam example tests to decrese execution time.

## How is this patch tested?

Diff:

|Name|Before|After|Diff.|
|:-|:-|:-|:-|
|hyperparam-params1|277s|217s|-60s|
|hyperparam-params2|158s|39s|-119s|
|hyperparam-params3|378s|89s|-289s|

Total: -468s

Before: https://github.com/mlflow/mlflow/runs/978722434

Screen Shot 2020-08-13 at 13 07 29

After: https://github.com/mlflow/mlflow/runs/978802148

Screen Shot 2020-08-13 at 13 19 15

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [x] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/server-infra`: MLflow server, JavaScript dev server
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: .github/workflows/examples.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
       run: |
         source ./dev/install-common-deps.sh
         source activate test-environment
-        pytest --verbose tests/examples --large
+        pytest --verbose tests/examples --durations=30 --large
```
Comment (harupy) [inline .github/workflows/examples.yml:42]: This overwrites `--durations` option specifed in `pytest.ini`.

Comment (harupy) [reply .github/workflows/examples.yml:42]: We have currently 22 example tests so 30 should be sufficient for a while.

Comment (mparkhe) [reply .github/workflows/examples.yml:42]: Is there a way to show durations for all tests by default? This way we do not need to actively manage this number as count of test increases?

**Review by mparkhe — APPROVED** [review]
Thanks @harupy. This is amazing!

File: .github/workflows/examples.yml (hunk 1 above)
Comment (harupy) [reply .github/workflows/examples.yml:42]: We could set `durations` to `0` which shows everything including setup and teardown.

**Pull request merged**